clients/admin-ui/src/features/dataset/DatabaseConnectForm.tsx

```diff
@@ -139,7 +133,7 @@ const DatabaseConnectForm = () => {
         error: string;
       }
     | {
-        classifyInstances: ClassifyInstanceResponseValues[];
+        classifyInstances: Record<string, string>[];
```

Comment:
This started erroring because of the change in type of `ClassifyDatasetResponse`. I tested it locally, and it turns out this does not return a `ClassifyInstanceResponseValues`. it returns a list that looks like 
```
[
  {
    dataset_name:  "bestpizzaco"
    id : "cls_f555ae90-25a7-4cdb-9d03-8bf644e6375f"
    status: "Created"
  },
...
]
```

I guess that's not explicitly typed in the backend though, so we just have `Record<string, string`. better than `any`, I guess?

Reply:
Sounds like a bug ticket for the API then, even if the fix is just to give a type definition to the route.

For now, `Pick<ClassifyInstanceResponseValues, 'dataset_name', 'id', 'status'>` might work?

Reply:
https://github.com/ethyca/fidesctl-plus/issues/331